cmd/sedad/cmd/config.go

```diff
@@ -81,3 +79,44 @@ func initTendermintConfig() *tmcfg.Config {
 
 	return cfg
 }
+
+func preUpgradeCmd() *cobra.Command {
```

Comment:
Do we have to worry about this command with future upgrades? Given that I don't see versions I guess it always runs.

Given that we set the `AllowUnencryptedSEDAKeys` to true in the migration this could potentially alter an operators config if they've set it to false.

Reply:
Yes good point.. Cosmovisor always runs it before applying an upgrade, so this will have to be removed in future upgrades.

Reply:
Or we add a version to the config file and check that to decide which migration to execute?

Reply:
I addressed the issue by just checking if seda config already exists here https://github.com/sedaprotocol/seda-chain/pull/549 I'd appreciate your review there 🙏